### ISSUE TITLE
No Aparezco en el Listado de Alumnos

### ISSUE THREAD

**Issue by CCancinoB** [IIC2233/Syllabus#33]
### Prerrequisitos
(Marcar colocando una X entre los corchetes los ítems que ya hiciste, así: "[X]")

* [X] Leí las reglas del foro (https://github.com/IIC2233/syllabus/issues/1).
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues).
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al Jefe de Bienestar o al correo del curso (iic2233@ing.puc.cl).
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla.

### Contenido

¡Hola! Cuando estuve presente en la Actividad Formativa 1, de la Sección 2 de este mismo curso, al pasar asistencia yo no estaba en el listado de alumnos para marcar si estaba 'presente'. A pesar de hacer CTRL+F de mi nombre completo, sólo apellido, sólo nombre, escrito de varias maneras, etc; sencillamente yo no estaba en el listado. Debido a esto, se me trató como ausente y el profesor me indicó que creara una issue en el Syllabus. ¿Alguna ayudita?

**Comment by cegalleta** [comment]
Hola!! 

Yo era el ayudante de la lista, finalmente te encontré y te dejé anotado!! Era un problema de mi compu.
No te preocupes :)

**Comment by andresmoraga0** [comment]
A mi me ocurrió lo mismo! Estoy en la sección 2

**Comment by cegalleta** [comment]
Sí!! Es que en el ctrl-F sólo detectaba cuando ponías los nombres en orden, pero era algo de mi compu. Si tienen el curso en canvas entonces sí o sí están inscritos!
